CMakeLists.txt

```diff
@@ -143,6 +143,9 @@ list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_PREFIX}/ll
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
 
+set_var_to_condition(MIOPEN_OFFLINE_COMPILER_PATHS_V2_DEFAULT FALSE)
+option(MIOPEN_OFFLINE_COMPILER_PATHS_V2 "Use rocm-core to find offline GPU compiler" ${MIOPEN_OFFLINE_COMPILER_PATHS_V2_DEFAULT})
```

Comment:
The PR will break Windows entirely. The ROCm-Core does not exist on Windows and will not be ported. It is available only on Linux.

Reply:
@apwojcik I think this won't break Windows unless MIOPEN_OFFLINE_COMPILER_PATHS_V2 is manually set to On, isn't it?

Reply:
@atamazov I will try compiling the branch on Windows tomorrow (CET) and fix issues, if any.

Reply:
I am confirming - it is compiling on Windows. Later, I will run the tests to verify the passing ratio.